src/piadm.sh

```diff
@@ -206,6 +206,12 @@ config_check() {
 		return
 	fi
 
+        if grep -q us-east.manta.joyent.com $PIADM_CONF ; then
+            # If they have a broken config with the stale joyent name,
+            # nuke it.
+            rm "${PIADM_CONF:?}"
+        fi
```

Comment:
I'm not entirely sure we should proactively delete it. SmartOS folks will need to delete it themselves before it will work again. Triton users may not ever know the difference.

I'm still soft inclined to keep it for Triton users.

The down side of this is that if someone wants to use us-east.manta.joyent.com, we're explicitly not letting them. Also, maybe this should be more targeted?

Reply:
IMHO you should be verbose about it, even without -v.  ANd nuke it.

Reply:
Still waiting on your opinion of verbosity.  Please defend lack thereof, or make this verbose.

Reply:
Updated to print a warning to stderr.